app/src/main/java/com/example/user/my_calculator/MainActivity.java

```diff
@@ -12,290 +12,132 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    Button btnNumber0;
-    Button btnNumber1;
-    Button btnNumber2;
-    Button btnNumber3;
-    Button btnNumber4;
-    Button btnNumber5;
-    Button btnNumber6;
-    Button btnNumber7;
-    Button btnNumber8;
-    Button btnNumber9;
-
-    TextView txtResult;
-
-    EditText edtInput;
-
-    Button btnCE;
-    Button btnC;
-    Button btnDelete;
+    private TextView txtResult;
```

Comment:
we don't use short value of names like txt/edt etc
Always give clean simple name without such things

Reply:
why not just name it resultOfOperation or somethink like that